codebase-manager/plugins/plugin.php

```diff
@@ -25,7 +29,7 @@ public function display_version_update_information( int $number_of_columns ): vo
 		}
 
 		printf( '<tr class="plugin-update-tr%s">', $this->is_active() ? ' active' : '' );
-		printf( '<td colspan="%s" class="plugin-update colspanchange">', esc_attr( $number_of_columns ) );
+		printf( '<td colspan="%s" class="plugin-update colspanchange%s">', esc_attr( $number_of_columns ), $this->has_vulnerabilities() ? ' hide-box-shadow' : '' );
```

Comment:
Why not just use `%d` or `%u` for `colspan`?

Reply:
No particular reason, maybe copy/pasta being the main one 😆. Ultimately it does end up being a string, but I can swap it.